cmd/devp2p/dnscmd.go

```diff
@@ -97,8 +97,8 @@ var (
 )
 
 const (
-	rootTTL     = 1
-	treeNodeTTL = 2147483647
+	rootTTL     = 30 * 60              // 30 min
+	treeNodeTTL = 2 * 7 * 24 * 60 * 60 // 2 weeks
```

Comment:
Any reason to decrease this? AFAIK this thing is immutable.

Reply:
I just thought the previous one was ridiculously high. Most DNS cache won't keep things for this long anyway.

Reply:
It was the highest that Cloudflare allowed. I know they won't keep it forever, but the current code will just cause everyone to rerequest everything every 2 weeks.